app/workers/notification_worker.rb

```diff
@@ -24,7 +24,7 @@ def perform(notification_params)
 
     @tags_hash  = Hash(notification_params[:tags])
     @links_hash = Hash(notification_params[:links])
-    lists = (lists_matched_on_links + lists_matched_on_tags).uniq { |l| l.id }
+    @document_type = notification_params[:document_type]
     delivery_ids = lists.map(&:gov_delivery_id)
 
     if lists.any?
```

Comment:
`lists` is now a method so we're calling this twice. It might be worth memoizing this as operations like `#uniq` can be quite expensive – usually O(n log n).